.dockerignore

```diff
@@ -0,0 +1,20 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+.Python
+env
+pip-log.txt
+pip-delete-this-directory.txt
+.tox
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.log
+.git
```

Comment:
¿Maybe .git* or /.git* ? (to include .gitignore)

Reply:
Sure, as we saw in class, every file or folder which are not needed to run the app should be ignored, those include `.git` folder, and even, `Dockerfile`, `.dockerignore`, etc.